lib/glific/partners.ex

```diff
@@ -610,6 +611,22 @@ defmodule Glific.Partners do
     )
   end
 
+  @doc """
+  Determine if we need to show roles and permission for an organization
+  """
+  @spec get_roles_and_permission(map()) :: boolean
+  def get_roles_and_permission(organization),
```

Comment:
```suggestion
  def is_roles_and_permission_enabled?(organization),
```

Reply:
Or let me know what is the purpose of set_roles_and_permission

Reply:
@pankaj-ag removed redundant code and changed names of functions